docs/Releases.md

```diff
@@ -13,6 +13,45 @@ use a date based versioning system.  Now, a release version can look
 like `17.11` where the "major" number is the year and the "minor" number
 is the month.
 
+## v21.09 (09/2020): Bug Fixes, Test cases, Dev Environment Improvements
+This release focused on general bug fixing and improving our test/development environments.
+A CloudLab template will be available with the latest release here: [https://www.cloudlab.us/p/GWCloudLab/onvm](https://www.cloudlab.us/p/GWCloudLab/onvm)
+
+**New Features and NFs**
+
+- [[243](https://github.com/sdnfv/openNetVM/pull/243)] Adds L3 Switch example based on DPDK `l3fwd` sample code. This NF can forward packets either using longest prefix match or a hash table lookup. 
+- [[254](https://github.com/sdnfv/openNetVM/pull/254)] Adds Fair Queue NF that demonstrates how to use advanced rings mode to directly access packets and distribute them to a set of child NFs. Packets are "classified" using a CRC32 hash and assigned to a queue. Queues are then read from in Round Robin order to process packets of different types in a fair way. Contributed by  ([Rohit M P](https://github.com/rohit-mp)) from NITK. 
+- [[277](https://github.com/sdnfv/openNetVM/pull/277)] Adds support for Jumbo frame packets. Enable by adding a `-j` flag to the manager's go.sh script.
+
+
+**Testing and Development Improvements**
+
+- [[296](https://github.com/sdnfv/openNetVM/pull/296)] Adds unit test for NF messaging infrastructure and fixes memory leak related to overflow of message pools [[Issue 293](https://github.com/sdnfv/openNetVM/pull/293)]. 
+- [[297](https://github.com/sdnfv/openNetVM/pull/297)] Adds VS Code profile to simplify debugging of NFs. 
+- [[302](https://github.com/sdnfv/openNetVM/pull/302)] Adds NF chain performance test to measure and plot inter-NF throughput and latency.
+
+
+**Miscellaneous Bug and Documentation Fixes**
+
+- [[304](https://github.com/sdnfv/openNetVM/pull/304)]
+- [[300](https://github.com/sdnfv/openNetVM/pull/300)]
+- [[270](https://github.com/sdnfv/openNetVM/pull/270)]
```

Comment:
```suggestion
- [[270](https://github.com/sdnfv/openNetVM/pull/270)] Fixes issues with relative path in the onvm go script to find the web directory. Now using `$ONVM_HOME` instead of `..`.
```